src/docs/schema.dox

```diff
@@ -187,9 +187,7 @@ to \c scanf, and takes a list of pointers to values in the same order:
 @snippet ex_all.c Get the cursor's record number key
 @snippet ex_all.c Get the cursor's composite key
 
-Cursors for a table have the same value format as the table, unless a
```

Comment:
This was removed due to git problems. It is part of [wt-12472](https://jira.mongodb.org/browse/WT-12472), not this PR. Put back.